src/components/UI/atoms/Image/Image.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { borderRadius } from '<variables>/border';
+import { boxShadows } from '<variables>/shadows';
+import { borderColors } from '<variables>/colorPalette';
+
+/**
+ * @description - Image Component
+ *
+ * @prop {string} width - image width
+ * @prop {string} height - image height
+ * @prop {string} imageUrl - image source
+ * @prop {string} altText - alt text
+ * @prop {string} boxShadow - shadowType
+ * @prop {boolean} borderColor- borderColor
+ * @prop {string} borderRadiusType - borderRadiusType
+ * @return {component} Image
+ */
+
+const Image = (
+  {
+    imageUrl, altText, width, height, boxShadow, borderRadiusType, borderColor,
+  },
+) => {
+  imageUrl = imageUrl || 'https://via.placeholder.com/150';
+  altText = altText || 'place holder image';
+
+  return <Image.Container
+    src={imageUrl}
+    alt={altText}
+    width={width}
+    height={height}
+    boxShadow={boxShadow}
+    borderRadiusType={borderRadiusType}
+    borderColor={borderColor}
+  />;
+};
+Image.Container = styled.img`
+${({
+    width,
+    height,
+    boxShadow,
+    borderRadiusType,
+    borderColor,
+    theme,
+  }) => {
+    return `
+  width: ${width} || '100%';
```

Comment:
unless you are using a boolean to conditionally render please use default props to initialize to 100%